madgiraffestudios/src/components/About/About.scss

```diff
@@ -11,7 +16,11 @@
      }
 }
 
-#footer-logo{
-    margin-bottom: 45px;
-}
+.img-fluid{
```

Comment:
should i give id's to these images?

Reply:
Only if it's required for some usage

Reply:
- [ ] can you name this something different, it's a reserved keyword for css class. Otherwise all images with be forced to use this setting

Reply:
understood addressed